src/alcaramba/Drawable/Town/Town.tscn

```diff
@@ -43,8 +48,30 @@ margin_right = 185.0
 margin_bottom = 231.0
 script = ExtResource( 2 )
 
+[node name="ContainerRight" type="VBoxContainer" parent="."]
+material = SubResource( 3 )
+margin_left = 695.0
+margin_top = 314.0
+margin_right = 1050.0
+margin_bottom = 912.0
+
+[node name="CurrentTile" type="VBoxContainer" parent="ContainerRight"]
+margin_right = 355.0
+margin_bottom = 50.0
+rect_min_size = Vector2( 0, 50 )
+
+[node name="SpareTiles" type="VBoxContainer" parent="ContainerRight"]
+unique_name_in_owner = true
+margin_top = 54.0
+margin_right = 355.0
+margin_bottom = 104.0
+rect_min_size = Vector2( 50, 50 )
+script = ExtResource( 4 )
+
```

Comment:
Can you please add the functionality to directly place a tile from the Market to the spare yard?

Reply:
out of scope